examples/lowMach/lowMach.oudf

```diff
@@ -1,3 +1,17 @@
+// 'device function' to return real gas properties
+// as functions of temperature
+void usrRealGasProperty(dfloat& T, dfloat& rho, dfloat& visc, dfloat& cond, dfloat& Cp)
```

Comment:
This is just an optional convenience function. If needed the user can define it. Plrease remove